test/pool.test.ts

```diff
@@ -738,6 +738,30 @@ describe("ReceiverWeights", function () {
     assert(weights.length == 0);
   });
 
+  it("Allows adding items after removing all items", async function () {
+    // Add an item and then clear the list
+    const weights_test = await deployReceiverWeightsTest();
+    const [addr1] = randomAddresses();
+
+    await weights_test.setWeights([
+      {receiver: addr1, weight: 1},
+      {receiver: addr1, weight: 0},
+    ]);
+
+    assert((await weights_test.receiverWeightsSumDelta()).eq(1 - 1));
```

Comment:
```suggestion
    assert((await weights_test.receiverWeightsSumDelta()).eq(0));
```

Reply:
This explains, where the value comes from. Here it's trivial, but it follows a pattern, which in other tests prevents unexpected values from popping up. I think that it makes no harm here, but it preserves the convention.